person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    @age >= 18 || @parent_permission
+  end
```

Comment:
- [ ] [OPTIONAL] You did well here, but if you look properly `@age >= 18` can be represented by the code at line 14 (`of_age?`). Kindly make use of it because it is a private function that can't be used elsewhere apart from its class definition.